packages/integrations/react/context.js

```diff
@@ -0,0 +1,21 @@
+const contexts = new WeakMap();
+
+export function getContext(result) {
+	if (contexts.has(result)) {
+		return contexts.get(result);
+	}
+	let ctx = {
+		c: 0,
+		get id() {
+			return 'r' + this.c.toString();
```

Comment:
What are `c` and `"r"`? Sorry for being pedantic, but I think we can have better names.

Like, we can assign `'r'` to a constant and give it a good name, so we know what it's that for.

Reply:
resolution: renamed `c` to `currentIndex`. assigned `'r'` to `const ID_PREFIX = 'r'` and interpolated.